packages/experiments/src/components/Form/index.d.ts

```diff
@@ -0,0 +1,15 @@
+export * from "./Form";
```

Comment:
hmm. Why is this a d.ts? This would produce no javascript...

Reply:
Good catch, thanks. Should be .ts